core/src/main/scala/cats/instances/partialOrdering.scala

```diff
@@ -0,0 +1,19 @@
+package cats
+package instances
+
+import cats.functor.Contravariant
+
+trait PartialOrderingInstances {
+  implicit val catsFunctorContravariantForPartialOrdering: Contravariant[PartialOrdering] =
+    new Contravariant[PartialOrdering] {
```

Comment:
can't this be contravariantCartesian using left to right ordering?

Reply:
Yup. Initially I was uncertain about adding this but ended up going for it.